src/devSVG.cpp

```diff
@@ -306,6 +319,10 @@ void svg_line(double x1, double y1, double x2, double y2,
   write_style_end(svgd->file);
 
   fputs(" />\n", svgd->file);
+
+  fputs("</svg>\n", svgd->file);
```

Comment:
@noamross Since there were changes made to several functions in `svglite`, I'll try to rewrite #46 soon and see how you can work on top of that. Thanks!